mm2src/coins/qrc20/swap.rs

```diff
@@ -113,46 +113,52 @@ impl Qrc20Coin {
         secret_hash: Vec<u8>,
         amount: BigDecimal,
         expected_swap_contract_address: H160,
-    ) -> Result<(), String> {
+    ) -> Result<(), MmError<ValidatePaymentError>> {
         let expected_swap_id = qrc20_swap_id(time_lock, &secret_hash);
-        let status = try_s!(
-            self.payment_status(&expected_swap_contract_address, expected_swap_id.clone())
-                .await
-        );
+        let status = self
+            .payment_status(&expected_swap_contract_address, expected_swap_id.clone())
```

Comment:
Please refactor `payment_status` for QRC20 to return `UtxoRpcError`

Reply:
done